genomel.bionimbus.org/manifest.json

```diff
@@ -9,12 +9,12 @@
   "versions": {
     "arborist": "quay.io/cdis/arborist:1.0.0",
     "fence": "quay.io/cdis/fence:2.6.0",
-    "indexd": "quay.io/cdis/indexd:1.0.10",
-    "peregrine": "quay.io/cdis/peregrine:1.0.2",
+    "indexd": "quay.io/cdis/indexd:1.1.1",
+    "peregrine": "quay.io/cdis/peregrine:1.1",
     "pidgin": "quay.io/cdis/pidgin:1.0.0",
     "revproxy": "quay.io/cdis/nginx:1.15.5-ctds",
-    "sheepdog": "quay.io/cdis/sheepdog:1.1.2",
-    "portal": "quay.io/cdis/data-portal:2.2.3",
+    "sheepdog": "quay.io/cdis/sheepdog:1.1.8",
+    "portal": "quay.io/cdis/data-portal:2.6.0",
```

Comment:
This portal version requires configuration changes or else the portal will break
without those it can only be 2.5.5